live-examples/js-examples/object/object-entries.html

```diff
@@ -8,7 +8,12 @@
 // expected output: Array ["2", "c"]
 
 const object3 = { 100: 'a', 2: 'b', 7: 'c' };
```

Comment:
With the changes suggested below made, you can remove this line.

Reply:
line 10 ?

Reply:
Yes